pkg/bucketclass/reconciler.go

```diff
@@ -209,11 +209,6 @@ func (r *Reconciler) ReconcilePhaseVerifying() error {
 			fmt.Sprintf("NooBaa system %q not found or deleted", r.NooBaa.Name))
 	}
 	if r.BucketClass.Spec.PlacementPolicy != nil {
-		numTiers := len(r.BucketClass.Spec.PlacementPolicy.Tiers)
```

Comment:
where do you add the call for the validation tests that were removed in this file?

Reply:
10 lines above, with:
```
err := ValidateBucketClass(r.BucketClass)
if err != nil {
    return util.NewPersistentError("ValidationError", err.Error())
}
```

Reply:
the validation call was already there, I just needed to remove the validations from the reconciler and add them to `bucketclass.ValidateBucketClass()`